edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/EdgeAgentConnection.cs

```diff
@@ -404,18 +409,18 @@ Task UpdateDeploymentConfig(TwinCollection desiredProperties)
         async Task<bool> WaitForDeviceClientInitialization() =>
             await Task.WhenAny(this.initTask, Task.Delay(DeviceClientInitializationWaitTime)) == this.initTask;
 
-        internal static bool CheckIfTwinSignatureIsValid(TwinCollection twinDesiredProperties)
+        internal bool CheckIfTwinSignatureIsValid(TwinCollection twinDesiredProperties)
         {
             // This function call returns false only when the signature verification fails
             // It returns true when there is no signature data or when the signature is verified
-            if (!CheckIfTwinPropertiesAreSigned(twinDesiredProperties))
+            if (!this.CheckIfManifestSigningIsEnabled(twinDesiredProperties))
             {
-                Events.TwinPropertiesAreNotSigned();
+                Events.ManifestSigningIsNotEnabled();
```

Comment:
If the user has put in an integrity section in the twin, but is missing the Manifest trust bundle - Is that more serious than the user not setting an integrity section? I am wondering if in this case, we might want to at least bump this up to a warning.

Reply:
Nevermind - I got the logic wrong.